diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter5/ParameterNameInOuterLambdaRule.kt

```diff
@@ -35,6 +50,19 @@ class ParameterNameInOuterLambdaRule(configRules: List<RulesConfig>) : DiktatRul
             )
         }
     }
+
+    /**
+     * ParameterNameInOuterLambdaConfiguration used when we need to allow the usage of 'it' in outer lambda
+     *
+     * @param config - map of strings with configuration options for a Parameter Name In Outer Lambda rule
+     */
+    class ParameterNameInOuterLambdaConfiguration(config: Map<String, String>) : RuleConfiguration(config) {
+        /**
+         * Flag (when false) allows to use `it` in outer lambda, if in inner lambdas would be no `it`
+         */
+        val strictMode = (config["strictMode"] == "true")
```

Comment:
Maybe it would be better to change it to `config["strictMode"]?.toBoolean() ?: true` or `config["strictMode"]?.toBoolean() ?: false` to be able to specify default value?

Reply:
Done